Bottom: a6d5cf5f447e422f2e5fa0301efc7c20f5a386e5
Top:    4c1992842aa0428834fb578c8dc279d3154fc1ac
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-21 21:18:17 +0000

Refresh of drawing-api-refactor

---

diff --git a/src/gui_draw.c b/src/gui_draw.c
index 1a3e608..cf0cb29 100644
--- a/src/gui_draw.c
+++ b/src/gui_draw.c
@@ -1,6 +1,7 @@
 #include "global.h"
 #include "data.h"
 #include "polygon.h"
+#include "hid_draw.h"
 
 static void
 fill_contour (PLINE *pl)
